packages/token-bridge-sdk/src/hooks/useArbTokenBridge.ts

```diff
@@ -399,24 +397,23 @@ export const useArbTokenBridge = (
     updateEthBalances()
   }
 
-  async function depositEthEstimateGas({
-    amount,
-    l1Signer
-  }: {
-    amount: BigNumber
-    l1Signer: Signer
-  }) {
-    const estimatedL1Gas = await gasEstimator.ethDepositL1Gas({
+  async function depositEthEstimateGas({ amount }: { amount: BigNumber }) {
+    // TODO: question - is from going to be walletAddress?
+    const depositRequest = await ethBridger.getDepositRequest({
       amount,
-      l1Signer,
-      l2Provider: l2.provider
+      from: walletAddress
     })
 
-    const {
-      maxGas: estimatedL2Gas,
-      maxSubmissionCost: estimatedL2SubmissionCost
-    } = await gasEstimator.ethDepositL2Gas(l2.provider)
+    const estimatedL1Gas = await l1.provider.estimateGas({
+      data: depositRequest.txRequest.data,
+      from: depositRequest.txRequest.from,
+      value: depositRequest.txRequest.value
+    })
 
+    const estimatedL2Gas = BigNumber.from(0)
+
+    // TODO: can we still get estimatedL2SubmissionCost in v3?
+    const estimatedL2SubmissionCost = BigNumber.from(0)
```

Comment:
Both `estimatedL2SubmissionCost` and `estimatedL2Gas` should be 0 for ETH deposits